Pod/Classes/RZAssert.h

```diff
@@ -397,12 +397,12 @@
 
 #define RZASSERT_CLASS_SUBCLASS_OF_CLASS(Subclass, Superclass) \
     do { \
-        RZASSERT_BASE( ([(Subclass) isSubclassOfClass:(Superclass)]), @"**** Bad Subclass Relationship **** \nReason: Expected class: \"%@\" to be a subclass of class: \"%@\", but it is not.", Subclass, Superclass ) \
+        RZASSERT_BASE( ([([Subclass class]) isSubclassOfClass:([Superclass class])]), @"**** Bad Subclass Relationship **** \nReason: Expected class: \"%@\" to be a subclass of class: \"%@\", but it is not.", [Subclass class], [Superclass class] ) \
```

Comment:
Parens aren't needed around the method calls `[Subclass class]` and `[Superclass class]`